ow_plexil/src/plexil-adapter/OwlatInterface.cpp

```diff
@@ -109,3 +293,552 @@ void OwlatInterface::owlatStowAction (int id)
      default_action_feedback_cb<ARM_STOWFeedbackConstPtr> (opname),
      default_action_done_cb<ARM_STOWResultConstPtr> (opname));
 }
+
+void OwlatInterface::owlatArmMoveCartesian (int frame, bool relative, 
+                                            vector<double> position, 
+                                            vector<double> orientation,
```

Comment:
`const vector<double>&` would be better here and in the other functions.  Otherwise, the vectors are copied.